wordle-web/layouts/default.vue

```diff
@@ -3,10 +3,50 @@
     <v-app-bar fixed app>
       <router-link to="/" class="text-h4 font-weight-black no-decoration">
         Wordle
+      </router-link>
+          <router-link to="/" class="  no-decoration">
+          <v-icon dark color="primary" x-large> mdi-magnify </v-icon>
       </router-link>
       <v-spacer />
       <settings-dialog />
+      <v-app-bar-nav-icon @click.stop="drawer = !drawer"></v-app-bar-nav-icon>
     </v-app-bar>
+
+      <v-navigation-drawer
```

Comment:
This looks really nice! I would suggest considering splitting this out into it's own component.